added test class for CheckFingerprint

a-filipp0v opened the pull request:
### Description

Added:
- CheckFingerprintTest class;
- fingerprinting_detail_metrics.example & fingerprinting_summary_metrics.example files in /testdata/picard/fingerprints as correct examples.

### Checklist (never delete this)

Never delete this, it is our record that procedure was followed. If you find that for whatever reason one of the checklist points doesn't apply to your PR, you can leave it unchecked but please add an explanation below.

#### Content
- [ ] Added or modified tests to cover changes and any new functionality
- [ ] Edited the README / documentation (if applicable)
- [ ] All tests passing on Travis

#### Review
- [ ] Final thumbs-up from reviewer
- [ ] Rebase, squash and reword as applicable

For more detailed guidelines, see https://github.com/broadinstitute/picard/wiki/Guidelines-for-pull-requests

yfarjoun commented on src/test/java/picard/fingerprint/CheckFingerprintTest.java:
```diff
@@ -0,0 +1,108 @@
+                "H=" + SUBSETTED_HAPLOTYPE_DATABASE_FOR_TESTING
+        };
+        Assert.assertEquals(runPicardCommandLine(args), 0);
+        Assert.assertEquals(checkResult(
```
I think it would be better to have two asserts, one for each metrics file....that way when something breaks it's easier to understand where the probem is

a-filipp0v replied:
Done!

yfarjoun commented on src/test/java/picard/fingerprint/CheckFingerprintTest.java:
```diff
@@ -0,0 +1,108 @@
+                new File(TEST_DATA_DIR + "/tempCheckFPDir/detail")), true);
+    }
+
+    @Test(dataProvider = "badData", expectedExceptions = {TribbleException.MalformedFeatureFile.class,
```
is the fully qualified class name needed?

a-filipp0v commented:
Hello, thanks for your feedback!
Made few changes in accordance with your comments:
- removed checkResult method, it’s invocation has been replaced by two asserts, one for each metrics file;
- full names of exceptions were replaced by short ones, corresponding imports added;
- few cosmetic changes, removed unnecessary throws from methods’ signature.

a-filipp0v commented:
I'm sorry, had some troubles with git branch, so all my small local commits were shown. Have made force push with amend, which replaced all of them with the last one. 

All mentioned changes are inside.

Sorry for inconvenience.
